app/views/articles/_reaction_button.html.erb

```diff
@@ -1,17 +1,21 @@
-<button
-  id="reaction-butt-<%= category %>"
-  aria-label="<%= aria_label %>"
-  aria-pressed="false"
-  class="crayons-reaction crayons-reaction--<%= category %>"
-  data-category="<%= category %>"
-  title="<%= description %>">
-  <span class="crayons-reaction__icon crayons-reaction__icon--inactive">
-    <%= crayons_icon_tag(image_path, aria_hidden: true) %>
-  </span>
-  <% if user_signed_in? # We cannot trigger the action state unless the user is signed in, so no need to render. %>
-  <span class="crayons-reaction__icon crayons-reaction__icon--active">
-    <%= crayons_icon_tag(image_active_path, aria_hidden: true) %>
-  </span>
-  <% end %>
+<div class="crayons-reaction">
+  <button
+    id="reaction-butt-<%= category %>"
+    aria-label="<%= aria_label %>"
+    aria-pressed="false"
+    class="crayons-reaction crayons-reaction--<%= category %> c-btn c-btn--icon-alone crayons-tooltip__activator"
```

Comment:
I think we will need to steer away from using the `c-btn` classes. I was experimenting with the reaction buttons and we seem to have introduced some funky layout shift as you press them:

https://user-images.githubusercontent.com/20773163/179960677-c3ec7a99-5e6a-4394-a098-41be6f540cc1.mp4

I'm assuming this is related to crayons button styles (I guess we deliberately deviated here). I'm pretty sure you can safely apply the `crayons-tooltip__activator` class without the `c-btn c-btn--icon-alone` as long as you add the `relative` class to help position the tooltip content - 

```suggestion
    class="crayons-reaction crayons-reaction--<%= category %> relative crayons-tooltip__activator"
```
Locally, that seems to fix the layout shift for me, and I think it would mean you could remove some of the other crayons workarounds you had to add (e.g. the background transparency)